uber/models.py

```diff
@@ -1805,12 +1805,18 @@ def track_pageview(cls, url, query):
             if "registration" in url:
                 with Session() as session:
                     attendee = session.query(Attendee).filter(Attendee.id == params['id']).first()
-                    Tracking.track(c.PAGE_VIEWED, attendee)
+                    if attendee is None:
```

Comment:
I believe this should be "if not attendee". That is more a style change than anything though.